README.md

```diff
@@ -37,16 +37,19 @@ Available options:
 * __host__ - InfluxDB host. `127.0.0.1` by default.
 * __protocol__ - `http` or `udp` for operating with InfluxDB. `http` by default.
 * __port__ - InfluxDB port. `8086` by default.
-* __db__ - database on InfluxDB for writing data. `exometer` by default
-* __username__ - username for authorization on InfluxDB.
-* __password__ - password for authorization on InfluxDB.
-* __timestamping__ - enable timestamping, `false` by default.
-* __tags__ - list of default tags for each data point. The `host` is automatically included here. 
+* __db__ - Database on InfluxDB for writing data. `exometer` by default
+* __username__ - Username for authorization on InfluxDB.
+* __password__ - Password for authorization on InfluxDB.
+* __timestamping__ - Enable timestamping, `false` by default. To enable `timestamping` with the reporter you can use `true` or `{true, Precision}` where `Precision` is a unit taken from `[n,u,ms,s,m,h]`. The default unit is `u`.
 * __batch_window_size__ - set window size in ms for batch sending. This means reported will collect measurements within this interval and send all measurements in one packet. `0` by default. 
 
-Timestamping is by default done by influxdb itself. To enable `timestamping` with the reporter you can use `true` or `{true, Precision}` where `Precision` is a unit taken from `[n,u,ms,s,m,h]`. The default unit is `u`.
+The following options can be set globally in the application config or locally in a specific subscription. The latter case overwrites the first.
```

Comment:
Why do we set it globally in application config? Can we set these like other options in reporter config?